config/config.go

```diff
@@ -939,6 +943,9 @@ func (c *Config) Valid() error {
 	if c.PreparedPlanCache.MemoryGuardRatio < 0 || c.PreparedPlanCache.MemoryGuardRatio > 1 {
 		return fmt.Errorf("memory-guard-ratio in [prepared-plan-cache] must be NOT less than 0 and more than 1")
 	}
+	if c.MemQuotaStatistics < DefMinQuotaStatistics {
```

Comment:
What if the value is `-1`? shall we provide an approach to disable this LRU feature?

Reply:
I 'll fix this